src/drivers/github.js

```diff
@@ -276,6 +282,17 @@ class Github {
     });
   }
 
+  get sha() {
+    if (GITHUB_EVENT_NAME === 'pull_request')
+      return github.context.payload.pull_request.head.sha;
+
+    return GITHUB_SHA;
+  }
+
+  get branch() {
+    return GITHUB_REF;
+  }
+
   get user_email() {
```

Comment:
does this need updating?

Reply:
branch?
If is branch the question, yes, we are needing it due to a special GH case (PR sha) that CML was actually managing.
I want to do a technical debt PR fixing that. CML.js not knowing drivers details so it gets the driver.branch not doing specific code which always smells

```js
const { sha } = get_driver(this);
if (sha) return sha;

  return await exec(`git rev-parse HEAD`);
```